lib/jekyll/configuration.rb

```diff
@@ -17,7 +17,7 @@ class Configuration < Hash
       # Handling Reading
       "safe"              => false,
       "include"           => [".htaccess"],
-      "exclude"           => %w(node_modules vendor),
+      "exclude"           => %w(node_modules vendor/bundle vendor/ruby vendor/cache),
```

Comment:
is there a way to use wildcards to exclude any future sub-directory under `/vendor/`